agent/browser/ie/wptbho/wptbho.cc

```diff
@@ -95,6 +97,24 @@ STDMETHODIMP_(void) WptBHO::OnDocumentComplete(IDispatch *pDisp,
 }
 
 /*-----------------------------------------------------------------------------
+Called when navigation is done
+-----------------------------------------------------------------------------*/
+STDMETHODIMP_(void) WptBHO::OnNavigateComplete(IDispatch *pDisp,
+  VARIANT * vUrl) {
+  CString url;
+  if (vUrl)
+    url = *vUrl;
+  AtlTrace(_T("[WptBHO] OnNavigateComplete - %s"), url);

```

Comment:
Should we put this inside the check for unknown_browser == unknown_frame? That way we don't have logging for other iframes